pf-recommended-practices/bp-guide-gh/ch1-model-formulation.md

```diff
@@ -63,8 +63,9 @@ back on
 A complete discussion of how to use the above rules in this context is outside
 the scope of this best-practice guide, but we offer a highly abbreviated
 discussion of the ''flavor,'' following the ideas of irreversible
-thermodynamics (we largely are following works like deGroot and Mazur
-{cite}`Groot2013`, although the continuum mechanics community
+thermodynamics (we recommend consulting works
+<!-- we largely are following works --> like deGroot and Mazur
```

Comment:
Commented text garbles PDF output; not sure if this is an issue for web rendering of Book

Reply:
I removed this comment, but doesn't impact website